config/initializers/bigquery.rb

```diff
@@ -0,0 +1,14 @@
+require 'google/cloud/bigquery'
+
+if FeatureFlag.active?(:send_request_data_to_bigquery)
+  BIG_QUERY_API_JSON_KEY = ENV['BIG_QUERY_API_JSON_KEY']
+
+  if BIG_QUERY_API_JSON_KEY.present?
+    Google::Cloud::Bigquery.configure do |config|
+      config.project_id  = ENV.fetch('BIG_QUERY_PROJECT_ID')
+      config.credentials = JSON.parse(BIG_QUERY_API_JSON_KEY)
+    end
+
+    Google::Cloud::Bigquery.new(project: ENV.fetch('BIG_QUERY_PROJECT_ID'))
```

Comment:
Haven't we just configured this at line 8?

Reply:
:+1: https://googleapis.dev/ruby/google-cloud-bigquery/latest/Google/Cloud/Bigquery.html#new-class_method
```
Parameters:

    project_id (String) (defaults to: nil) —

    Identifier for a BigQuery project. If not present, the default project for the credentials is used.
```

so no need to pass `project_id` twice here

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4601/files#diff-5265f727a65a1861aa971a561450e9d062a12e37cd8b21531a1ab4721bb85f71R11